apps/condo/domains/user/integration/sberid/sync/syncUser.js

```diff
@@ -66,6 +66,7 @@ const syncUser = async ({ context, userInfo, userType }) => {
     const userIdentities = await UserExternalIdentity.getAll(context, {
         identityType: SBER_ID_IDP_TYPE,
         identityId: userInfo.id,
+        deletedAt: null,
```

Comment:
It looks strange. Can we add a todo to remove this parameter? We should by default have only not deleted objects

Reply:
ok